ddtrace/internal/datadog/profiling/crashtracker/_crashtracker.pyx

```diff
@@ -0,0 +1,171 @@
+    service_bytes = ensure_binary_or_empty(service)
+    crashtracker_set_service(string_view(<const char*>service_bytes, len(service_bytes)))
+
+
+@not_implemented
+def set_env(env: StringType) -> None:
+    env_bytes = ensure_binary_or_empty(env)
+    crashtracker_set_env(string_view(<const char*>env_bytes, len(env_bytes)))
+
+
+@not_implemented
+def set_version(version: StringType) -> None:
+    version_bytes = ensure_binary_or_empty(version)
+    crashtracker_set_version(string_view(<const char*>version_bytes, len(version_bytes)))
+
+
+@not_implemented
+def set_runtime(runtime: StringType) -> None:
+    runtime_bytes = ensure_binary_or_empty(runtime)
+    crashtracker_set_runtime(string_view(<const char*>runtime_bytes, len(runtime_bytes)))
+
+
+@not_implemented
+def set_runtime_version(runtime_version: StringType) -> None:
+    runtime_version_bytes = ensure_binary_or_empty(runtime_version)
+    crashtracker_set_runtime_version(string_view(<const char*>runtime_version_bytes, len(runtime_version_bytes)))
+
+
+@not_implemented
+def set_library_version(library_version: StringType) -> None:
+    library_version_bytes = ensure_binary_or_empty(library_version)
+    crashtracker_set_library_version(string_view(<const char*>library_version_bytes, len(library_version_bytes)))
+
+
+@not_implemented
+def set_stdout_filename(filename: StringType) -> None:
+    filename_bytes = ensure_binary_or_empty(filename)
+    crashtracker_set_stdout_filename(string_view(<const char*>filename_bytes, len(filename_bytes)))
+
+
+@not_implemented
+def set_stderr_filename(filename: StringType) -> None:
+    filename_bytes = ensure_binary_or_empty(filename)
+    crashtracker_set_stderr_filename(string_view(<const char*>filename_bytes, len(filename_bytes)))
+
+
+@not_implemented
+def set_alt_stack(alt_stack: bool) -> None:
+    crashtracker_set_alt_stack(alt_stack)
+
+
+@not_implemented
+def set_resolve_frames_disable() -> None:
+    crashtracker_set_resolve_frames_disable()
+
+
+@not_implemented
+def set_resolve_frames_fast() -> None:
+    crashtracker_set_resolve_frames_fast()
+
+
+@not_implemented
+def set_resolve_frames_full() -> None:
+    crashtracker_set_resolve_frames_full()
+
+
+@not_implemented
+def set_resolve_frames_safe() -> None:
+    crashtracker_set_resolve_frames_safe()
+
+
+@not_implemented
+def set_profiling_state_sampling(on: bool) -> None:
+    if on:
+        crashtracker_profiling_state_sampling_start()
+    else:
+        crashtracker_profiling_state_sampling_stop()
+
+
+@not_implemented
+def set_profiling_state_unwinding(on: bool) -> None:
+    if on:
+        crashtracker_profiling_state_unwinding_start()
+    else:
+        crashtracker_profiling_state_unwinding_stop()
+
+
+@not_implemented
+def set_profiling_state_serializing(on: bool) -> None:
+    if on:
+        crashtracker_profiling_state_serializing_start()
+    else:
+        crashtracker_profiling_state_serializing_stop()
+
+
+@not_implemented
```

Comment:
Should this be removed?

Reply:
I don't think so, but I do think the choice of the name for the decorator was quite poor.  I've factored this to be called `raise_if_unimplementable` in b559de2f9